holoviews/plotting/bokeh/element.py

```diff
@@ -389,7 +395,12 @@ def _axis_properties(self, axis, key, plot, dimension=None,
         Returns a dictionary of axis properties depending
         on the specified axis.
         """
-        axis_props = {}
+        try:
+            axis_props = (built_in_themes[self.renderer.theme]
```

Comment:
If we'll be doing this a lot maybe we should have a utility to get the default properties for a particular model type.